_posts/0000-01-02-chrishussein.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
sorry im not in the moood to be whitty right now, been on this thing alllllll day